### DISCUSSION TITLE
Should we "white label" the admin login screen?

### DISCUSSION THREAD

**Discussion by jonyo** [geodesicsolutions-community/geocore-community#41] [Q&A]
I was just thinking about this as I was working on updating the copyright.

Technically the admin is viewable to everyone unless you do something special to block even viewing the admin login.

And admin/ is not that secret of a URL, it is easy to guess.

Think we should remove product name and any links to the website on the login itself?

The only downside is that it would require removing the link to "forgot password" that links to the wiki article about how to reset the admin pass.  But maybe that is fine, since site owners will know what software it is and can look it up?

What is your thought, leave it as-is, or should we remove branding and just say `Admin Panel Login v. 20.0.0` and that's it?

**Comment by vicos59** [comment]
That's a common issue with any off-the-shelf software.

As a matter of standard practice, I always change the admin directory or script name (whatever the software permits) and change the admin user to something other than 'admin'

Bad guys gonna be bad guys. With so much to do already, I wouldn't worry about this. Just point out good security practices in the docs.

**Reply by jonyo** [reply]
If we want to update the name in the login, we'll already need to do some amount of work.  It is non-trivial because the name is an image not text.  So I thought, while at it, why not just replace it with generic "Admin Panel Login" and remove the reset password link.

**Reply by vicos59** [reply]
I would do everything to the Keep It Simple standard. Eliminate unnecessary graphics; people can use CSS if they really want something fancy for a login page. If you really want to keep it simple, make it generic as possible and remove any reference to 'admin.' Not sure if the bad guys search for things like that when they're up to no good. The people who have a right to be there know what it is for.
